### DISCUSSION TITLE
Tags?

### DISCUSSION THREAD

**Discussion by jcm4atx** [ArchiveBox/ArchiveBox#590] [Q&A]
I see the ability to filter my archive by tags. What I can't figure out is how to add/edit tags to my entries.

**Comment by cdvv7788** [comment]
In the admin there is a field to do that

**Reply by jcm4atx** [reply]
When you say "in the admin", what exactly do you mean? I can get to a page that lists Id, Url, Timestamp, etc. but nothing there is editable.

**Reply by cdvv7788** [reply]
You can create a superuser with `archivebox manage createsuperuser` and then start the server with `archivebox server`. You can see the admin list in `/admin`. I think this needs some documentation.

**Reply by pirate** [reply]
What version are you on? v0.4.21 doesn't support tag editing via the UI, but v0.4.24 and up do.

**Reply by jcm4atx** [reply]
Ah ha!

ArchiveBox v0.4.21

I do plan to upgrade, but I also have the question, is it possible to add/edit tags from the command line? And once I figure this out I'm happy to help with the documentation.

**Comment by jcm4atx** [comment]
I solved my own problem.

I had performed a pip install which installed a down-rev version (0.4.21) of the tool. I removed that and installed the docker-compose version which is at 0.5.0.

My reverse proxy setup was still valid, so that piece still works. And the GUI is vastly improved.

**Reply by cdvv7788** [reply]
Did you have any issue with the index update?

**Reply by jcm4atx** [reply]
I wasn't sure how that was supposed to work, so I didn't try. I pointed the docker install to another directory. I had few items in my initial install and much of that was garbage. I was getting a feel for how the software worked. I pulled the files out of my original ./sources directory, deleted what I didn't want, and rebuilt my archive.

If I had had a lot in there I would have put effort into the reindex. But since it was on the order of dozens I didn't.
